src/mask.js

```diff
@@ -188,6 +189,9 @@ angular.module('ui.mask', [])
                             if (angular.isDefined(iAttrs.uiMaskPlaceholderChar)) {
                                 iAttrs.$observe('uiMaskPlaceholderChar', initPlaceholderChar);
                             }
+                            iAttrs.$observe('uiUnsafe', function(unsafe){
```

Comment:
I feel like you should check if the attribute is defined before `$observe`ing it similar to the other attribute options.